### PR TITLE
PR between 5359343-20230720 <-- aefd43e-20230728

### PR DESCRIPTION
**Pull request by tashrifbillah**
Gnar's company's last work was `Share Form #334` . All further commits were for fixing DPdash deployment issues faced at MGB server due to our VM's minimal CPU configuration. This PR will serve as a visualization of changes.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,10 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>DPdash</title>
-  </head>
+  <script defer src="/js/bundle.js"></script></head>
```
Comment (tashrifbillah) [inline public/index.html:8]: Did you move `<script>` element from body to head? @Elkrival

File: server/app.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ if (process.env.NODE_ENV === 'development') {
   app.use(connectLiveReload())
 }
 /** favicon setup */
-app.use(favicon(path.join(__dirname, '../public/favicon.ico')))
+app.use(favicon(path.join(__dirname, '../public/img/favicon.ico')))
```
Comment (tashrifbillah) [inline server/app.js:52]: Why was this change necessary?

File: public/index.html (hunk 1 above)
Comment (Elkrival) [reply public/index.html:8]: Webpack appends this during the build process, it recreates an index.html file using the template file and it routes to the built JavaScript.

File: server/app.js (hunk 2 above)
Comment (Elkrival) [reply server/app.js:52]: We should have a directory structure that is specific to where things are. It makes sense for images to go in an img directory. This was how it was before as well.
